05-Jason-Luo/wk07/1-mon/GoogleBooks/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>GoogleBooks</title>
+  </head>
+  <body>
+    <h1>GoogleBooks Catalogue</h1>
+
+    <input name="title" id="bookList" placeholder="Title" required />
+    <button id="button" type="button">Search books</button>
+
+    <div id="display"></div>
+
+    <!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script> -->
+    <script src="js/googlebooks.js"></script>
```

Comment:
Next time have a go at putting the script tag in the head and using the document ready event listener:
```js
document.addEventListener('DOMContentLoaded', () => {
    // code goes here
});
```
It's standard practice to keep your script tags in your html `<head>`.